### PR TITLE
PRED-907 --pred-name documentation and positive class changes

### PR DESCRIPTION
**Pull request by sevikkk**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,3 +1,17 @@
+X.X.X (?)
```
Comment (sevikkk) [inline CHANGES.rst]: it's 1.8.7-rc, that hopefully becomes master today

**Comment by pprett** [comment]
LGTM :+1:

**Comment by Axik** [comment]
can we also put doc update here https://github.com/datarobot/batch-scoring#running-batch_scoring ?

I know that this is duplication, that we might fix. But this is how it works currently.

**Comment by mckaymatt** [comment]
Looks good

**Pull request merged**